tests/integration_tests/test_visualization.py

```diff
@@ -292,6 +293,10 @@ def test_visualization_compare_classifiers_from_prob_csv_output_saved(
                     ground_truth,
                     '--output_feature_name',
                     output_feature_name,
+                    '--split_file',
+                    split_file,
+                    '--ground_truth_metadata',
```

Comment:
would probably call it training_set_metadata or just metadata isntead of ground_truth_metadata

Reply:
Quick question...the `ludwig visualization` cli has a pre-existing parameter by the name of `ground_truth_metadata`
![image](https://user-images.githubusercontent.com/1425269/97793334-00251380-1bc1-11eb-9a88-e5a4df92f181.png)

From your perspective, how does this `ground_truth_metadata` differ with `training_set_metadata`?

Since we are retrieving the prediction raw values from the predictions csv files, I need a way to translate the string values to the encoded numeric values since many of the visualization functions assume use of the encoded numeric representation.

Reply:
I think it's just a name choice, we can stick to ground truth metadata for consistency.